src/app/issues/PageDefaultIssues.tsx

```diff
@@ -0,0 +1,270 @@
+import { ScopeTag } from '@/app/scopes/ScopeTag';
+import {
+  DataList,
+  DataListCell,
+  DataListRow,
+  Icon,
+  useToastError,
+} from '@/components';
+import { RouterInput } from '@/server/routers/_app';
+import { trpc } from '@/utils/trpc';
+
+export const PageDefaultIssues = () => {
+  const [selectedIssues, setSelectedIssues] = useState<DefaultIssue[]>([]);
+
+  const navigate = useNavigate();
+
+  const isMobile =
+    useBreakpointValue({
+      base: true,
+      md: false,
+    }) ?? false;
+
+  const issues = defaultIssues.map((issue) => ({
+    ...issue,
+    scopes: defaultScopes.filter((scope) => issue.scopes.includes(scope.id)),
+  }));
+
+  const trpcContext = trpc.useContext();
+  const { mutate: scopeMutation, isLoading: isScopeLoading } =
+    trpc.scope.createMany.useMutation();
+  const { mutate: issueMutation, isLoading: isIssueLoading } =
+    trpc.issue.createMany.useMutation();
+
+  const toastError = useToastError();
+
+  const onSubmit = () => {
+    const scopeIds = [
+      ...new Set(selectedIssues.flatMap((issue) => issue.scopes)),
+    ];
+    const scopesToAdd: RouterInput['scope']['createMany'] = defaultScopes
+      .filter((scope) => scopeIds.includes(scope.id))
+      .map((scope) => ({
+        id: scope.id,
+        name: scope.name,
+        description: scope.description ?? null,
+        color: scope.color ?? null,
+      }));
+
+    const issuesToAdd: RouterInput['issue']['createMany'] = selectedIssues.map(
+      (issue) => ({
+        id: issue.id,
+        title: issue.name,
+        description: issue.description ?? null,
+        scopes: issue.scopes,
+      })
+    );
+
+    scopeMutation(scopesToAdd, {
+      onSuccess: () => {
+        trpcContext.scope.all.invalidate();
+
+        return issueMutation(issuesToAdd, {
+          onSuccess: () => {
+            navigate(-1);
+
+            return trpcContext.issue.all.invalidate();
+          },
+          onError: () => {
+            toastError({
+              title: 'Failed to create issue',
+            });
+          },
+        });
+      },
+      onError: () => {
+        toastError({
+          title: 'Failed to create scope',
+        });
+      },
+    });
+  };
+
+  const isLoading = isScopeLoading || isIssueLoading;
+
+  const handleSelectedIssuesChanged = (
+    checked: boolean,
+    issue: DefaultIssue
+  ) => {
+    if (checked) {
+      setSelectedIssues([
+        ...selectedIssues.filter(
+          (_issue: DefaultIssue) => _issue.id !== issue.id
+        ),
+        issue,
+      ]);
+    } else {
+      setSelectedIssues(
+        selectedIssues.filter((_issue: DefaultIssue) => _issue.id !== issue.id)
+      );
+    }
+  };
+
+  console.log({ issues });
+
+  return (
+    <Page containerSize="lg" isFocusMode>
+      <PageTopBar showBack onBack={() => navigate(-1)}>
+        <Heading size="md">{t('issues:defaults.title')}</Heading>
+      </PageTopBar>
+      <PageContent>
```

Comment:
UX of the page can be improve related to the functionality, with your version the user select issues but the main stuff is the scope and that what the user should choose in first. 
We can say that on scope selection, all defined suggested issues for the scopes are displayed and automatically selected but the user can unselected each issue.